src/ResourceManager/Compute/Commands.Compute/Commands.Compute.csproj

```diff
@@ -64,6 +64,9 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.KeyVault.2.3.0-preview\lib\net452\Microsoft.Azure.Management.KeyVault.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.Azure.Management.Network, Version=16.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@pashukl Please revert all changes to files that are not .cs files.

Reply:
@pashukl At this point, especially since you have a small number of changes, the easiest method would be to delete the branch and create a new PR based on an up to date fork.